tests/misc_env/install_prereq.py

```diff
@@ -410,6 +413,8 @@ def registry_login(ceph, distro_ver):
         file.write(json.dumps(auths_dict, indent=4))
         file.flush()
         file.close()
+    # import pdb
```

Comment:
can you please remove this commented code?

Reply:
Done Sunil